pkgs/applications/blockchains/bchd/default.nix

```
@@ -0,0 +1,23 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+buildGoModule rec {
+  pname = "BCHD";
```

```suggestion
  pname = "bchd";
```

see https://nixos.org/manual/nixpkgs/stable/#sec-package-naming